.gitignore

```diff
@@ -11,6 +11,7 @@
 /log/
 !/cache/.gitkeep
 !/log/.gitkeep
+*.un~
```

Comment:
yeah. my bad. i thought it might be a good addition back then.

but now i can easily see how my nom-standard (although highly advised) undo vim configuration should not be reflected here.

in fact, instead of adding a global git ignore, i moved them all into my .vim folder, as i should have done since the beginning!

now i will also need to increase my garbage collector plan... 😏

anyway, already removed it for the next commit. thanks!